src/gsi/gsimod.F90

```diff
@@ -2038,20 +2038,6 @@ subroutine gsimain_initialize
      baldiag_inc =.false.
   end if
 
-! If reflectivity is intended to be assimilated, beta_s0 should be zero.
-  if ( beta_s0 > 0.0_r_kind )then
-    ! skipped in case of direct reflectivity DA because it works in Envar and hybrid
-    if ( l_use_rw_columntilt .or. l_use_dbz_directDA) then
-       do i=1,ndat
-          if ( if_model_dbz .and. (index(dtype(i), 'dbz') /= 0) )then
-             write(6,*)'beta_s0 needs to be set to zero in this GSI version, when reflectivity is directly assimilated. &
-                        Static B extended for radar reflectivity assimilation will be included in future version.'
-             call stop2(8888)
-          end if
-       end do
-    end if
-  end if
-
```

Comment:
This removal is simply to prevent from stopping radar reflectivity DA in the setting of hybrid EnVar.

The static B is not extended for reflectivity DA for now, so the static B for reflectivity is regarded as zero in the setting of hybrid EnVar. This is not necssarily the best for reflectivity DA, but zero static-B (ignoring the impact of reflectivity DA in the static-B part) itself is not so strange setting. In addition, the influence of zero static-B is not so large in case that the weight of ensemble B is much larger than that of static B.

Extending static B for reflectivity is defenitely important but it is not included in this PR.

Reply:
Since static B is not extended for reflectivity DA yet, assimilating reflectivity with static B in the hybrid EnVar doesn't sound right scientifically, even though the impact might not be that big. Do you have test figures to show the difference, say beta_s0=0.15 vs beta_s0=0 when only assimilating reflectivity in the hybrid EnVar?
And if you have to remove the check here, is it possible to add a check somewhere else?

Reply:
I compared the analysis increment of beta_s0=0 and beta_s0=0.15. Could you see the example of analysis increment shown in [PR-543_DBZ-DA-hybrid.pptx](https://github.com/NOAA-EMC/GSI/files/11255340/PR-543_DBZ-DA-hybrid.pptx)? In the test of beta_s0=0.15, the analysis increment is slightly smaller due to smaller ensemble B and substantial zero static B, but it looks reasonable and realistic.

> And if you have to remove the check here, is it possible to add a check somewhere else?

I added the warning of zero static B for reflectivity instead of fully removing this part. Does that solve your concern?

Reply:
Thanks for comparing the analysis and adding the warning message - this information is important. I am okay with the change and will approve the PR now.

Reply:
> 
> The static B is not extended for reflectivity DA for now, so the static B for reflectivity is regarded as zero in the setting of hybrid EnVar. This is not necssarily the best for reflectivity DA, but zero static-B (ignoring the impact of reflectivity DA in the static-B part) itself is not so strange setting. In addition, the influence of zero static-B is not so large `in case that the weight of ensemble B is much larger than that of static B.`
> 
@shoyokota What about the case the ensemble weight is 0.5 (3DRTMA used this setting in some situations) or smaller? 
A general question, is it possible to use a different ensemble weight for different variables?

Reply:
@guoqing-noaa The smaller ensemble weight, the smaller impact of reflectivity DA (the closer to the situation of zero-B for the variables not included in static-B). The weight is fixed in the current code, but I think it is possible to make the weight changeable.

Reply:
Thanks, @shoyokota 
It will be good if we can still be able to use 100% EnVar for Ref and similar variables while doing hybrid EnVar for other variables in one step.

Reply:
@guoqing-noaa In the current GSI code, the implementation is not so simple. If we apply pure EnVar only except for variables in static-B (sf,vp,t,ps), cross-variable covariance between (sf,vp,t,ps) and the other variables becomes larger, which affects not only the radar reflectivity DA but also the conventional DA. It is out of the objective of this PR and not tested at all. Even currently, humidity is not included in the variables in static-B, so I believe zero static-B for hydrometeors is currently the most straightforward way also for reflectivity. It is very interesting to introduce the changeable weight for different variables, but it is better to think about it in another PR after evaluated carefully.

Reply:
> @guoqing-noaa In the current GSI code, the implementation is not so simple. If we apply pure EnVar only except for variables in static-B (sf,vp,t,ps), cross-variable covariance between (sf,vp,t,ps) and the other variables becomes larger, which affects not only the radar reflectivity DA but also the conventional DA. It is out of the objective of this PR and not tested at all. Even currently, humidity is not included in the variables in static-B, so I believe zero static-B also for the other variables including hydrometeors is currently the most straightforward way. It is very interesting to introduce the changeable weight for different variables, but it is better to think about it in another PR after evaluated carefully.

I agree. Thanks, @shoyokota